src/app.ts

```diff
@@ -0,0 +1,70 @@
+import { Orderbook } from '@0x/asset-swapper';
+import { WSClient } from '@0x/mesh-rpc-client';
+import * as express from 'express';
+
+import * as defaultConfig from './config';
+import { SRA_PATH } from './constants';
+import { getDBConnectionAsync } from './db_connection';
+import { logger } from './logger';
+import { requestLogger } from './middleware/request_logger';
+import { OrderBookServiceOrderProvider } from './order_book_service_order_provider';
+import { OrderWatcherService } from './services/order_watcher_service';
+import { OrderBookService } from './services/orderbook_service';
+import { SRAHttpService } from './services/sra_http_service';
+import { StakingDataService } from './services/staking_data_service';
+import { StakingHttpService } from './services/staking_http_service';
+import { SwapHttpService } from './services/swap_http_service';
+import { SwapService } from './services/swap_service';
+import { WebsocketService } from './services/websocket_service';
+import { OrderStoreDbAdapter } from './utils/order_store_db_adapter';
+import { providerUtils } from './utils/provider_utils';
+
+// tslint:disable:no-unused-expression
+/**
+ * returns the app
+ */
+export async function getAppAsync(config: any = defaultConfig): Promise<express.Application> {
```

Comment:
might be worth thinking about how to apply this pattern to the other "apps" in the `runners/` directory.